apps/bfd-server/bfd-server-war/src/test/java/gov/cms/bfd/server/war/r4/providers/TransformerTestUtilsV2.java

```diff
@@ -113,6 +115,15 @@ static void assertEobCommonClaimHeaderData(
     }
   }
 
+  /**
+   * Gets the golden beneficiary id.
+   *
+   * @return the golden beneficiary id
+   */
+  static String getGoldenBeneId() {
+    return GOLDEN_BENE_ID;
+  }
+
```

Comment:
Is there a better place anyone can think of to put this? It seems ok here, but realistically we need it outside this package too. If there is some common util that multiple packages could read from to get this data, this might be better placed there?

Reply:
maybe in a class under gov.cms.bfd.model.rif.samples since that is where the data lives too? Do you think we need the method though or can we just go with a single constant?